src/components/interfaces/MOBILE_API.xml

```diff
@@ -3346,6 +3362,12 @@
                 True: Available, False: Not Available, Not present: Not Available.
             </description>
         </param>
+        <param name="climateEnableAvailable" type="Boolean" mandatory="false" since="5.1">
```

Comment:
since="5.2"?

Reply:
@yang1070 Fixed in 4d29747d10ac9d7d7d4294f7a7be4c35ee132c75